src/python/director/footstepsdriver.py

```diff
@@ -90,35 +95,47 @@
                                  'Max Downward Step': 0.18,
                                  'Behavior': 2,
                                  'Leading Foot': 0,
-                                 'Swing Height': 0.03,
-                                 'Drake Swing Speed': 0.6,
-                                 'Drake Instep Shift': 0.005,
-                                 'Drake Min Hold Time': 1.0,
+                                 'Support Contact Groups': 0,
+                                 'Prevent Swing Undershoot': 0,
+                                 'Prevent Swing Overshoot': 0,
+                                 'Map Mode': 0},
+                       'IHMC Nominal': {'Min Num Steps': 0,
+                                 'Max Num Steps': 16,
+                                 'Min Step Width': 0.20,
+                                 'Nominal Step Width': 0.26,
+                                 'Nominal Forward Step': 0.26,
+                                 'Max Forward Step': 0.30,
+                                 'Max Step Width': 0.32,
+                                 'Max Upward Step': 0.18,
+                                 'Max Downward Step': 0.18,
+                                 'Behavior': 2,
+                                 'Leading Foot': 0,
                                  'Support Contact Groups': 0,
                                  'Prevent Swing Undershoot': 0,
                                  'Prevent Swing Overshoot': 0,
                                  'Map Mode': 0}}
-DEFAULT_STEP_PARAMS['Terrain'] = DEFAULT_STEP_PARAMS['Drake Nominal'].copy()
-DEFAULT_STEP_PARAMS['Terrain'].update({'Drake Min Hold Time': 1.0,
-                                       'Drake Swing Speed': 0.6,
-                                       'Swing Height': 0.05,
+
+DEFAULT_PLANNING_PARAMS['Terrain'] = DEFAULT_PLANNING_PARAMS['Drake Nominal'].copy()
+DEFAULT_PLANNING_PARAMS['Terrain'].update({#'Drake Min Hold Time': 1.0,
+                                       #'Drake Swing Speed': 0.6,
+                                       #'Swing Height': 0.05,
```

Comment:
I'm not sure, but I think it is a bug to not include these parameters in this parameter set.

Thinking about this a little more, I'm not so sure any more that it adds value to separate planning and control parameters.  One could argue that the separation is a detail that should not be exposed to the user.  To the user, they are parameters that affect walking.  In theory, a planner may need to be told about swing speed in order to compute a correct plan.